backend/app/services/streaming/executor_streaming.py

```
@@ -0,0 +1,495 @@
+                await self._process_reasoning_chunk(state, data)
+            elif chunk_type == "workbench_delta":
+                await self._process_workbench_delta(state, data)
+            elif chunk_type == "status":
+                await self._process_status_chunk(db, state, data)
+            else:
+                logger.warning(f"Unknown chunk type: {chunk_type}")
+
+            # Perform periodic saves
+            await self._periodic_save(db, state)
+
+            return {"status": "success", "offset": state.offset}
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)